### PR TITLE
Dummy data init

### PR DESCRIPTION
**Pull request by cmcrow2**

### CODE REVIEW

**Review by kennyctran — DISMISSED** [review]
visibility int,  air_temp int,  water_temp int,  weight int, max_depth, start_pressure and end_pressure

to string

**Review by kennyctran — COMMENTED** [review]
Thanks,  @cmcrow2 looks good to me.

@acdavitt can you do a final pass through to double-check?

File: lib/dummyData/dummyData.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        time_out: new Date().toISOString(),
+        rnt: 20,
+        abt: 15,
+        tbt: 35,
+        start_pressure: '5 psi',
+        end_pressure: '10 psi',
+        notes: "I saw a sea turtle and a jellyfish!",
+        dive_center: "Big Island Divers",
+        dive_instructor: "Frank",
+        instructor_cert: "123456",
```
Comment (acdavitt) [inline lib/dummyData/dummyData.js:39]: @cmcrow2 For the instructor_cert in the dummy data, the cert number is a string, but in the schema it is an int.  Could we change above to varchar as well?

File: lib/db/schema.sql [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  time_out varchar(25),
+  rnt int,
+  abt int,
+  tbt int,
+  start_pressure varchar(20),
+  end_pressure varchar(20),
+  notes text,
+  dive_center varchar(50),
+  dive_instructor varchar(50),
+  instructor_cert int,
```
Comment (acdavitt) [inline lib/db/schema.sql]: @cmcrow2 See below, but can we change instructor_cert to varchar as well?

**Review by acdavitt — CHANGES_REQUESTED** [review]
instructor_cert: change from int to varchar?

**Review by acdavitt — APPROVED** [review]
Thanks!

**Pull request merged**